src/harrastuspassi/harrastuspassi/serializers.py

```diff
@@ -119,7 +128,8 @@ class Meta:
             'organizer',
             'permissions',
             'price_type',
-            'price_amount'
+            'price_amount',
+            'municipality'
```

Comment:
Code style recommends trailing commas in cases like this. One benefit is that in git you don't have to touch both lines (diff, history) and another is that it's easier to avoid syntax errors due to missing commas when copying and editing.